### PR TITLE
Supported Nothing Value

### PR DESCRIPTION
**Pull request by koudaiii**
## WHY

I'd like to create tag which designated only key.

ex:

```
$ ./ec2c  tag --instance i-12345abc --tags deploy-server
$ ./ec2c  tag --instance i-12345abc --tags deploy-server,role=app
```

### CODE REVIEW

File: command/tag.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (c *TagCommand) Run(args []string) int {
 	for _, tag := range strings.Split(tagString, ",") {
 		keyValue := strings.Split(tag, "=")
 
-		if len(keyValue) >= 2 {
+		if len(keyValue) >= 1 {
 			tags = append(tags, &ec2.Tag{
 				Key:   aws.String(keyValue[0]),
 				Value: aws.String(strings.Join(keyValue[1:], "=")),
```
Comment (dtan4) [inline command/tag.go:62]: Is this really safe?

How about this code:

``` go
var key string
var value string

if len(keyValue) == 1 {
  key, value = aws.String(keyValue[0]), aws.String("")
} else {
  key, value = aws.String(keyValue[0]), aws.String(strings.Join(keyValue[1:], "=")))
}
```

Comment (koudaiii) [inline command/tag.go:62]: I see!! Thank you for review!

**Comment by dtan4** [comment]
LGTM 👍

**Comment by koudaiii** [comment]
Thank you!!

**Pull request merged**